Index: src/kendo.popup.js
===================================================================
--- src/kendo.popup.js	(revision )
+++ src/kendo.popup.js	(revision )
@@ -3,6 +3,7 @@
         ui = kendo.ui,
         touch = kendo.support.touch,
         getOffset = kendo.getOffset,
+        bodyResizeEvent = $.browser.msie && $.browser.version < 9,
         OPEN = "open",
         CLOSE = "close",
         CENTER = "center",
@@ -23,6 +24,7 @@
         ACTIVECHILDREN = ".k-picker-wrap, .k-dropdown-wrap, .k-link",
         MOUSEDOWN = touch ? "touchstart" : "mousedown",
         SCROLLER = "kendoMobileScroller",
+        DOCUMENT= $(document),
         cssPrefix = kendo.support.transitions.css,
         TRANSFORM = cssPrefix + "transform",
         extend = $.extend,
@@ -112,7 +114,16 @@
             $(document.documentElement).bind(MOUSEDOWN, proxy(that._mousedown, that));
 
             if (!touch) { //  On mobile device this closes the popup if keyboard is shown
+                that._currentWidth = DOCUMENT.width();
                 $(window).bind("resize scroll", function() {
+                    if (bodyResizeEvent) {
+                        var width = DOCUMENT.width();
+                        if (width == that._currentWidth) {
+                            return;
+                        }
+                        that._currentWidth = width;
+                    }
+
                     if (!that._hovered) {
                         that.close();
                     }
